src/git/index.ml

```diff
@@ -1,345 +0,0 @@
-(*
```

Comment:
This was implementing the handling of `git checkout`. Have you re-implemented that as well?